docs/contributors/tests.mdx

```diff
@@ -61,10 +61,17 @@ Examples:
 
 A level 5 heading (`#####`), with the text being one of the following:
 
-- `##### only` -- If at least one test has the `only` annotation, the runner will run this/these test(s).
 - `##### skip` -- If a test has the `skip` annotation, the runner will not run that test.
 
-These are typically added to tests temporarily, so try not to commit tests with annotations. There must be either zero or one annotations in a test.
+This annotation is typically added to tests temporarily, so try not to commit tests with annotations. There must be either zero or one annotations in a test.
```

Comment:
I think `skip` is mostly used to mark the tests that don't work YET, but we have intention to make it pass in the future.

The usual usage should be: add `skip` annotations if your tests intended to work but don't either because it's not yet implemented or because of the existing bug. Make sure to add corresponding comment with explanation. If you need to temporarily include/exclude some tests see instructions below

Reply:
updated.